cestrosphendon

indoptogopt opened the pull request.

indoptogopt committed:
adding an article about the cestrosphendon

TheShadowOfHassen commented on gui/encyclopedia/articles/civilizations/mace/military/cestrosphendon.json:
```diff
@@ -0,0 +1,5 @@
+{
+    "title": "The Cestrosphendon",
+    "subtitle": "The forgotten weapon of Macedonia - a dart sling",
+    "content": "The cestrosphendon (or simply “cestrus”) was a special type of ancient missile weapon. The word stems from its ancient Greek name “kestrosphendone”, comprised of the words “kestros” referring to a military bolt or dart, and “sphendone” translating to “sling”. Unsurprisingly, that describes the essential idea behind it: combining the sheer power of the sling with the deadly penetra ...
```
```suggestion
    "content": "The cestrosphendon (or simply “cestrus”) was a special type of ancient missile weapon. The word stems from its ancient Greek name “kestrosphendone”, comprised of the words “kestros” referring to a military bolt or dart, and “sphendone” translating to “sling”. Unsurprisingly, that describes the essential idea behind it; combining the sheer power of the sling with the deadly penetration force of the bow and arrow -- by launching small darts off a modified sling.\n\nDisappointingly, this is where all certainty with our information ends. No specimen of this weapon has survived, or at least no finds could be identified as one. All the evidence we have is from literature, and very scarce. To be precise, there are only two proven text passages describing the cestrosphendon, one from Livy and one from Polybius. They mention the weapon being deployed by the Macedonian army under Perseus during the third Macedonian War against Rome.\n\nOf course, they also describe the weapon at hand, but do so only quite roughly. The passages are quite puzzling and do not provide a clear picture of what the cestrosphendon looked and worked like.\n\nSeveral reconstruction attempts have been made and shown interesting results. Although it should be noted that the findings are likely to some degree distorted by modern standards and expectations.\n\nIn tests, the cestrosphendon showed great performance in terms of damage, Roman legionaries certainly weren't safe at all from its darts in their armor - which is also something Livy underlined (despite the Roman army eventually emerging victorious). Potential downsides include awkward weapon handling and unreliability, which leads to the conclusion that a high level of training was required to wield the cestrosphendon. For example, difficulties arose with the placement of the dart on the sling. The angular motion of the swing would often cause an unwanted fishtailing (wobbling) behavior on the dart. Potential tricks around it were found, but in turn lengthened the reload time. And aiming proved quite hard as well.\n Even now, the existence and usage cestrosphendon remains a mystery. One that might never be fully solved. But isn't that what makes history so fascinating?"

TheShadowOfHassen reviewed (commented):
The edit is more of a content than anything. I changed the beginning of the second paragraph, fixed some punctuations and polished the ending to (in my opinion) make it flow smoother and give it some kick.

Accept the changes if you want and I'll look over this a second time sometime soon.

indoptogopt committed:
Update gui/encyclopedia/articles/civilizations/mace/military/cestrosphendon.json

Co-authored-by: TheShadowOfHassen <94244235+TheShadowOfHassen@users.noreply.github.com>

indoptogopt committed:
add missing line break

indoptogopt commented:
👍

indoptogopt commented:
> The edit is more of a content than anything. I changed the beginning of the second paragraph, fixed some punctuations and polished the ending to (in my opinion) make it flow smoother and give it some kick.
> 
> Accept the changes if you want and I'll look over this a second time sometime soon.

👍

TheShadowOfHassen commented on gui/encyclopedia/articles/civilizations/mace/military/cestrosphendon.json:
```diff
@@ -0,0 +1,5 @@
+{
+    "title": "The Cestrosphendon",
+    "subtitle": "The forgotten weapon of Macedonia - a dart sling",
+    "content": "The cestrosphendon (or simply “cestrus”) was a special type of ancient missile weapon. The word stems from its ancient Greek name “kestrosphendone”, comprised of the words “kestros” referring to a military bolt or dart, and “sphendone” translating to “sling”. Unsurprisingly, that describes the essential idea behind it; combining the sheer power of the sling with the deadly penetra ...
```
```suggestion
    "content": "The cestrosphendon (or simply “cestrus”) was a special type of ancient missile weapon. The word stems from its ancient Greek name “kestrosphendone”, comprised of the words “kestros” referring to a military bolt or dart, and “sphendone” translating to “sling”. Unsurprisingly, that describes the essential idea behind it; combining the sheer power of the sling with the deadly penetration force of the bow and arrow -- by launching small darts off a modified sling.\n\nDisappointingly, this is where all certainty with our information ends. No specimen of this weapon has survived, or at least no finds could be identified as one. All the evidence we have is from literature, and very scarce. To be precise, there are only two proven text passages describing the cestrosphendon, one from Livy and one from Polybius. They mention the weapon being deployed by the Macedonian army under Perseus during the third Macedonian War against Rome.\n\nOf course, they also describe the weapon at hand, but do so only quite roughly. The passages are quite puzzling and do not provide a clear picture of what the cestrosphendon looked and worked like.\n\nSeveral reconstruction attempts have been made and shown interesting results. Although it should be noted that the findings are likely to some degree distorted by modern standards and expectations.\n\nIn tests, the cestrosphendon showed great performance in terms of damage, and Roman legionaries certainly weren't safe at all from its darts in their armor - which is also something Livy underlined (despite the Roman army eventually emerging victorious). Potential downsides include awkward weapon handling and unreliability, which leads to the conclusion that a high level of training was required to wield the cestrosphendon. For example, difficulties arose with the placement of the dart on the sling. The angular motion of the swing would often cause an unwanted fishtailing (wobbling) behavior on the dart. Potential tricks around it were found, but in turn lengthened the reload time. And aiming proved quite hard as well.\n\n Even now, the existence and usage cestrosphendon remains a mystery. One that might never be fully solved. But isn't that what makes history so fascinating?"

TheShadowOfHassen reviewed (commented):
Last change, adding an 'and'

indoptogopt committed:
Update gui/encyclopedia/articles/civilizations/mace/military/cestrosphendon.json

Co-authored-by: TheShadowOfHassen <94244235+TheShadowOfHassen@users.noreply.github.com>

indoptogopt commented:
Sorry, I forgot about this PR.
Is it ready to be merged now?

TheShadowOfHassen commented:
Yes, looks good.
